com.unity.ml-agents/Runtime/SimpleMultiAgentGroup.cs

```diff
@@ -12,7 +12,9 @@ public class SimpleMultiAgentGroup : IMultiAgentGroup, IDisposable
         readonly int m_Id = MultiAgentGroupIdCounter.GetGroupId();
         HashSet<Agent> m_Agents = new HashSet<Agent>();
 
-
+        /// <summary>
+        /// Unregister all member Agents on dispose of SimpleMultiAgentGroup.
```

Comment:
This describes what the code does. I think it should just be `Disposes of the SimpleMultiAgentGroup`.